### PR TITLE
Removed `--input` flag

### PR DESCRIPTION
**Pull request by shikharsoni07**
- `--input` flag was removed as it was no longer needed after deletion of `data.txt` as middleware
- also removed one irrelevant comment

### CODE REVIEW

File: generate_ics.py
Comment (proffapt) [inline generate_ics.py:60]:
```suggestion
    Creates an ICS file `timetable.ics` with the timetable data present inside the 'timetable' parameter.
```

**Review by proffapt — COMMENTED** [review]
Look for other changes as well, which are related to removal of `data.txt`. 
Once confirmed, make the required changes if any.. will approve the PR once you confirm this.

**Review by proffapt — APPROVED** [review]
LGTM!

**Comment by proffapt** [comment]
@ashwinpra will wait for your review.

**Review by ashwinpra — APPROVED** [review]

**Comment by ashwinpra** [comment]
Looks good to me too, merging

**Pull request merged**
